### ISSUE TITLE
Two or more Datatables at the same time

### ISSUE THREAD

**Issue by kastoras** [singlequote/Laravel-datatables#13]
Hello,

I have problem when i try to have two datatables in the same Page.

Initialise datatables:
```
$companytable = \DataTable::model(\App\Company::class)->tableModel(\App\TableModels\Companies::class);

$producttable = \DataTable::model(\App\Product::class)->tableModel(\App\TableModels\Products::class);
```

Send to view:
`return view('view_file',[ ])->with(compact('companytable','producttable'));`

Try to use them on view file:
```
 {!! $companytable->table() !!}
 {!! $companytable->script() !!}

 {!! $producttable->table() !!}
 {!! $producttable->script() !!}
```

With this code i get Datatables error:

> DataTables warning: table id=laravelDataTable5d5d52687a111 - Ajax error.

Is it possible to do something like this or i can have only one datatable per page;

Thanks in advance.

**Comment by wimurk** [comment]
Hi @kastoras,

Can you show me the complete error trace of the datatable response?

**Comment by wimurk** [comment]
Hi @kastoras,

I reproduced the problem. The package can't handle 2 or more tables at onces. In the next version update the issue will be fixed.

**Comment by kastoras** [comment]
Ok thank you, if i will find a solution i will let you know.

**Comment by wimurk** [comment]
issue is fixed in #15 see the changes.

The problem is with the Facade. It overwrites the tables. You can use multiple tables by directly calling the package controller.

```php
use SingleQuote\DataTables\DataTable;

$companytable =(new DataTable)->model(\App\Company::class)->tableModel(\App\TableModels\Companies::class);

$producttable = (new DataTable)->model(\App\Product::class)->tableModel(\App\TableModels\Products::class);
```
